src/Arcus.Testing.Tests.Integration/Integration/DataFactory/RunDataFlowTests.cs

```diff
@@ -69,6 +70,51 @@ public async Task RunDataFlow_WithCsvFileOnSource_SucceedsByGettingCsvFileOnSink
             AssertCsv.Equal(expected, result.GetDataAsCsv(ConfigureCsv));
         }
 
+        [Fact]
+        public async Task RunDataFlow_WithCsvFileOnSourceAndDataSetParameter_SucceedsByGettingCsvFileOnSinkWithSubPathParameter()
+        {
+            // Arrange
+            IDictionary<string, string> sourceDataSetParameterKeyValues = new Dictionary<string, string>
+            {
+                { RandomizeWith("sourceDataSetParameterKey"), RandomizeWith("SourceDataSetParameterValue") },
+                { RandomizeWith("sourceDataSetParameterKey"), RandomizeWith("SourceDataSetParameterValue") }
+            };
+            IDictionary<string, string> sinkDataSetParameterKeyValues = new Dictionary<string, string>
+            {
+                { RandomizeWith("sinkDataSetParameterKey"), RandomizeWith("sinkDataSetParameterValue") }
+            };
+
+            await using var dataFlow = await TemporaryDataFactoryDataFlow.CreateWithCsvSinkSourceAsync(Configuration, Logger, ConfigureCsv, dataFlowOptions =>
+            {
+                dataFlowOptions.Source.SetSourceDataSetParameterKeyValues(sourceDataSetParameterKeyValues);
+                dataFlowOptions.Sink.SetSinkDataSetParameterKeyValues(sinkDataSetParameterKeyValues);
```

Comment:
Can these be called `AddFolderPathParameter` maybe? to clearly show it later in the `UploadToSource`, that these are sub paths.

Reply:
Fixed in [#67d3aa](https://github.com/arcus-azure/arcus.testing/pull/228/commits/67d3aaba825c61e6673e1b8e1653103268cebcdf)